src/editor/tests/test_user.py

```diff
@@ -0,0 +1,15 @@
+# from django.test import TestCase
```

Comment:
It's going to be moved to a `registration` app that I'm working on. I don't want the merge conflict or a temporary broken test.